code/setup.dm.back

```diff
@@ -785,6 +785,7 @@ var/list/be_special_flags = list(
 #define IS_SKRELL 3
 #define IS_UNATHI 4
 #define IS_XENOS  5
+#define IS_VAURCA 6
```

Comment:
This file change is worthless. .back means that said file is not edited. But since it's not compiled, it appears that the define is either never used, or applied properly elsewhere too.